### PR TITLE
Add order_by clause documentation for API v2

### PR DESCRIPTION
**Pull request by rkouye**
Add Order by clause documentation for API v2

### CODE REVIEW

File: source/includes/v2/_odsql.md [hunk 1]
```diff
@@ -804,3 +804,34 @@ The count of pattern letters determine the format.
 ##### Format:
 `group_by=date_format(<string_literal>)`
 in which `<string_literal>` contains a date format
+
+## Order by clause
+
+> Order by examples
+```python
```
Comment (vbarbaresi) [inline source/includes/v2/_odsql.md]: It doesn't look like Python though!

Comment (vbarbaresi) [reply source/includes/v2/_odsql.md]: But joke aside, looking at it again, it provides better highlighting than nothing and could be added for the other ODSQL snippets in this file (currently there are written as ``` plain-text )

That or  ``` mysql that also kinda works

Comment (rkouye) [reply source/includes/v2/_odsql.md]: Indeed! I tried SQL also and I like it even more. I will change it to this, and even add another PR to change it everywhere.

**Comment by baptabl** [comment]
@rkouye don't we want to also add the `order_by` in param tables of
https://github.com/opendatasoft/ods-documentation-explore-api/blob/73d4a2b4d3c7be8d8b634b8efd407ffdc01f48e1/source/includes/v2/_catalog.md#aggregating-datasets
and https://github.com/opendatasoft/ods-documentation-explore-api/blob/73d4a2b4d3c7be8d8b634b8efd407ffdc01f48e1/source/includes/v2/_dataset.md#aggregating-records?

**Pull request merged**